core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -697,6 +697,29 @@ class SQLPlanParserSuite extends BasePlanParserSuite {
     }
   }
 
+  test("approx_count_distinct is supported in aggregations") {
```

Comment:
The test overall is ok. A couple of comments here:

- lets put the actual expression in the test name `HyperLogLogPlusPlus` instead of  `approx_count_distinct`
- this test should succeed for non-DB environments accoring to https://github.com/NVIDIA/spark-rapids/issues/12388 . Although, we do not have unit-tests for databricks, we can at least add a comment on the unit test as a reminder that Databricks should fail that test until RAPIDS issue is fixed. The other option is to create an eventlog for databricks, or inject metadata for databricks to fool the tools in thinking that this is a databricks eventlog.